app/src/main/res/layout-sw600dp-port/topic_lessons_story_summary.xml

```diff
@@ -64,6 +64,7 @@
 
               <TextView
                 android:id="@+id/story_progress_percentage"
+                style="@style/TextViewCenter"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
```

Comment:
`layout_gravity` should be removed right?

Reply:
This is related to layout gravity and not the TextViews gravity. It is controlled by the parent layout. This is how it looks you can check in screenshot.
<img width="144" alt="Screenshot_1637662491" src="https://user-images.githubusercontent.com/16301028/143006595-201ee456-6bd1-4014-bd9d-7b2da06bb0ee.png">

Reply:
Ack.